backend/src/app/urls/account_urls.py

```diff
@@ -3,9 +3,16 @@
 from app.api.account import (
     get_account,
     get_all_account,
+    signup_account,
+    update_account,
+    delete_account
 )
 
 urlpatterns = [
     path('', get_all_account),
     path('<int:pk>', get_account),
+    path('signup', signup_account),
```

Comment:
In rest API convention for single model crude use different HTTP verb for different operation, not separate URL.
e.g 
GET api/v1/account ---> List all account
GET api/v1/account/<id> --> Get single account
POST api/v1/account --> Create account
PUT api/v1/account --> Update 
DELETE api/v1/account -- Delete account

Reply:
Got it, I will make the changes